### PR TITLE
E2e test

### PR DESCRIPTION
**Pull request by birom**
I set the development environment in the default proxy conf because of this issue: https://github.com/angular/angular-cli/issues/6358

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -53,3 +53,9 @@ deploy:
       && .ci-travis/push-version-bump.sh ${BUILD_VERSION} ${GIT_USER} ${GIT_PWD}
     on:
       branch: master
+  - provider: script
+    skip_cleanup: true
+    script: ".ci-travis/maven-e2e.sh"
+    on:
+      all_branches: true
+      condition: "$TRAVIS_EVENT_TYPE == cron"
```
Comment (birom) [inline .travis.yml]: We check here if the build is scheduled build eg:
Selection_164

File: alv-portal-ui/package.json [hunk 2]
```diff
@@ -3,10 +3,11 @@
...
     "ng": "ng",
-    "start": "npm run start:dev",
-    "start:dev": "npm run i18n && ng serve --base-href / --watch --proxy-config proxy.conf.json",
-    "start:mobile": "npm run i18n && ng serve --host 0.0.0.0 --base-href / --watch --proxy-config proxy.conf.json",
-    "start:prod": "npm run i18n && ng serve --base-href / --watch --prod --aot --source-map --proxy-config proxy.conf.json",
+    "start": "npm run start:staging",
+    "start:dev": "npm run i18n && ng serve --base-href / --watch --proxy-config proxy.conf.dev.json",
+    "start:staging": "npm run i18n && ng serve --base-href / --watch --proxy-config proxy.conf.staging.json",
+    "start:mobile": "npm run i18n && ng serve --host 0.0.0.0 --base-href / --watch --proxy-config proxy.conf.staging.json",
+    "start:prod": "npm run i18n && ng serve --base-href / --watch --prod --aot --source-map --proxy-config proxy.conf.staging.json",
```
Comment (ganqqwerty) [inline alv-portal-ui/package.json]: do we need source-maps here?

File: .travis.yml (hunk 1 above)
Comment (ganqqwerty) [reply .travis.yml]: ran we run it like hourly?

Comment (birom) [reply .travis.yml]: No unfortunatley not. Possible values are: daily, weekly and monthly.

File: alv-portal-ui/package.json (hunk 2 above)
Comment (birom) [reply alv-portal-ui/package.json]: I am not sure. I've only changed the ``proxy-config part``.

**Pull request merged**